ci/docker/runtime_functions.sh

```diff
@@ -924,6 +924,16 @@ nightly_straight_dope_python3_multi_gpu_tests() {
       test_notebooks_multi_gpu.py --nologcapture
 }
 
+# check commit for flaky tests
+select_tests(){
+    set -ex
+    python tool/flaky_test_bot/test_selector.py -b HEAD~1 HEAD
+}
+run_flakiness_checker(){
+    set -ex
+    python tools/flaky_test_bot/check_tests.py
```

Comment:
Please note that this is python2 opposed to python3 as you specifeid in the shebang. You can also omit the python entirely here